packages/gafl-webapp-service/src/pages/contact/address/lookup/address-lookup.njk

```diff
@@ -58,7 +56,7 @@
                   classes: "govuk-input--width-20",
                   autocomplete: 'given-name',
                   attributes: { maxlength: 50 },
-                  errorMessage: { text: 'Enter your house number or name' } if error['premises']
+                  errorMessage: { text: 'Enter the house number or name' } if error['premises']
```

Comment:
Inconsistent - we use "building number or name" above and "house number or name" here